src/components/app/details/triggerView/cdMaterial.tsx

```diff
@@ -628,10 +641,12 @@ export class CDMaterial extends Component<CDMaterialProps, CDMaterialState> {
                                 />
                             </>
                         ) : !_canReviewConfig ? (
-                            <>
-                                <WarningIcon className="no-config-found-icon icon-dim-16" />
-                                &nbsp; Config Not Available
-                            </>
+                            this.state.recentDeploymentConfig && (
```

Comment:
please Reduce ternary or use if else statement for better readability

Reply:
done